app/controllers/notifications_controller.rb

```diff
@@ -19,7 +19,7 @@ def update
   private
 
   def notification
-    @notification = Notification.first
+    @notification = Notification.first || Notification.create
```

Comment:
How about we use the `.first_or_create` method instead?

Reply:
yes, cool. I didn't know about that one. I only knew about find_or_create :)